docs/earthfile/earthfile.md

```diff
@@ -352,6 +380,34 @@ Instructs Earthly to not overwrite the file creation timestamps with a constant.
 
 Instructs Earthly to keep file ownership information.
 
+#### Examples
+
+Assuming the following directory tree, of a folder named `test`:
+
+```
+test
+  └── file
+
+```
+
+Here is how the following `SAVE ARTIFACT ... AS LOCAL` commands will behave:
+
+```
+save-artifact-example:
+    WORKDIR base
+    COPY test .
+
+    # This will copy the base folder into the output directory.
+    # You would find file at out-dot/base/file.
+    SAVE ARTIFACT . AS LOCAL out-dot/
+
+    # This will copy the contents of the base folder into the output directory.
+    # You would find sub-file at out-glob/file. Note the base directory is not in the output.
+    SAVE ARTIFACT ./* AS LOCAL out-glob/
```

Comment:
```suggestion
WORKDIR base
COPY test .

# This will copy the base folder into the output directory.
# You would find file at out-dot/base/file.
SAVE ARTIFACT . AS LOCAL out-dot/

# This will copy the contents of the base folder into the output directory.
# You would find sub-file at out-glob/file. Note the base directory is not in the output.
SAVE ARTIFACT ./* AS LOCAL out-glob/
```